deployments/deploy-ssh.sh

```diff
@@ -672,6 +674,10 @@ spec:
       value: "${NFS_PATH}"
     - name: SKIP_PX_OPERATOR_UPGRADE
       value: "${SKIP_PX_OPERATOR_UPGRADE}"
+    - name: CLOUD_CRED_CONFIG
```

Comment:
I do not know what are the uses of these 2 Environment Variable, but we should just accept the path to `CLOUD_CRED_CONFIG_JSON`
I will leave the code implementation to you guys.
@mkoppal-px